README.md

```diff
@@ -1 +1,72 @@
-# judgement-day
\ No newline at end of file
+#judgement-day
+***
+## Feel Free to Rate FACN2 Mentors
+***
+###How to use our Website:
+To check out the app, Please clone this Repository.
+```javascript
+git clone
+```
+Install all the Node dependencies
+```javascript
+
+```
+Run start
+```javascript
+
+```
+---
+##Our Product:
+We want to build a web application so the students of FACN2 can rate the mentors on categories and
```

Comment:
Maybe no need for the bit after 'can' because it is repeated on line 22, so this line could just be

' We want to build a web application so the students of FACN2 can: '